### PR TITLE
Added support to SMTP server for password recovery

### PR DESCRIPTION
**Pull request by stefano-daitan**

### CODE REVIEW

**Comment by stefano-daitan** [comment]
resolves ns1labs/orb.live/#3

**Comment by stefano-daitan** [comment]
resolves ns1labs/orb.live#3

File: charts/orb/templates/users-deployment.yaml [hunk 1]
```diff
@@ -65,6 +65,23 @@ spec:
...
+            - name: MF_EMAIL_USERNAME
+              value: {{ required "A valid .Values.orb.smtp.username is required!" .Values.orb.smtp.username | quote }}
+            - name: MF_EMAIL_PASSWORD
+              value: {{ required "A valid .Values.orb.smtp.password is required!" .Values.orb.smtp.password | quote }}
+            - name: MF_EMAIL_FROM_ADDRESS
+              value: {{ required "A valid .Values.orb.smtp.fromAddress is required!" .Values.orb.smtp.fromAddress | quote }}
+            - name: MF_EMAIL_FROM_NAME
+              value: {{ .Values.orb.smtp.fromName | quote }}
+            - name: MF_EMAIL_TEMPLATE
+              value: {{ .Values.orb.smtp.emailTemplatePath | quote }}
```
Comment (stefano-daitan) [inline charts/orb/templates/users-deployment.yaml:83]: Maybe I should hard-code "/email.tmpl" here...

Comment (weyrick) [reply charts/orb/templates/users-deployment.yaml:83]: Seems reasonable for now

Comment (weyrick) [reply charts/orb/templates/users-deployment.yaml:83]: Or else change those to be the variable

**Review by weyrick — APPROVED** [review]

File: charts/orb/templates/users-deployment.yaml (hunk 1 above)
Comment (stefano-daitan) [reply charts/orb/templates/users-deployment.yaml:83]: I'll hard-code this value for now.

**Pull request merged**